frameworks/helloworld/tests/test_region_awareness.py

```diff
@@ -14,18 +14,45 @@
 POD_NAMES = ['hello-0', 'world-0', 'world-1']
 
 
-@pytest.mark.dcos_min_version('1.11')
-@pytest.mark.region_awareness
-@sdk_utils.dcos_ee_only
-def test_nodes_deploy_to_local_region_by_default():
+@pytest.fixture
+def local_service():
+    try:
+        sdk_install.install(
+            config.PACKAGE_NAME,
+            config.SERVICE_NAME,
+            3,
+            additional_options={
+                "service": {"scenario": "MULTI_REGION", "allow_region_awareness": "true"}
+            })
+        sdk_plan.wait_for_completed_deployment(config.SERVICE_NAME)
```

Comment:
`sdk_install.install` already waits for completed deployment. Is there a specific reason to call this again?